web/src/routes/SurveyPages/SurveyListPage.tsx

```diff
@@ -143,11 +193,21 @@ interface Survey {
 export default function SurveyListPage() {
   const [theme, toggleTheme] = useTheme();
   const [surveys, setSurveys] = useState<Survey[]>([]);
+  const [page, setPage] = useState<number>(1);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isFetch, setIsFetch] = useState<boolean>(false);
   const navigate = useNavigate();
 
   const fetchSurveyData = async (): Promise<void> => {
-    const request: AxiosResponse<Survey[]> = await axios.get<Survey[]>(requests.fetchSurveyListPage + 1);
-    setSurveys(request.data);
+    setIsLoading(true);
+    // TODO: More detailed error handling is needed.
+    try {
+      const request: AxiosResponse<Survey[]> = await axios.get<Survey[]>(requests.fetchSurveyListPage + page);
```

Comment:
Okay. It's time to apply AbortController for axios.

Think about this case:

1. User clicks `page 1` button,
2. But for whatever reason, network is too slow or not that fast as expected.
3. The user changed her mind to navigate to `page 2`.
4. So she clicked `page 2` button (Before even page 1 is fully loaded)

In this case, at the moment she clicked `page 2` button, initial request to `page 1` should be cancelled.
(If the request isn't cancelled, you're making double requests to Server)
And the AbortController comes here. AbortController can make fetching API to be cancelled. 

So, please add `cancelToken` option for `axios.get()` request and catch error on `catch` function or `interceptor` if you want.